pkgs/by-name/un/undertime/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, python3Packages
+, fetchFromGitLab
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "undertime";
+  version = "4.0.0";
+  pyproject = true;
+
+  src = fetchFromGitLab {
+    owner = "anarcat";
+    repo = pname;
```

There is a preference to avoid overuse of `pname` (see #277994)

```suggestion
    repo = "undertime";
```

---

I don't necessarily agree with the necessity of it, but this is the second time I've been told this, so I'll go ahead and capitulate. ;)